lib/simctl.js

```diff
@@ -75,7 +75,11 @@ async function createDevice (name:string, deviceTypeId:string,
 }
 
 async function deleteDevice (udid:string):void {
-  await simExec('delete', 0, [udid]);
+  let loopFn:Function = async () => {
+    await simExec('delete', 0, [udid]);
+  };
+  // retry delete with a sleep in between because it's flakey
+  await retryInterval(5, 200, loopFn);
```

Comment:
Interesting. The error seems to be consistently a permissions thing:

```
An error was encountered processing the command (domain=NSCocoaErrorDomain, code=513):\n“FC850CAE-F401-4C6F-A15F-A7E021656755” couldn’t be removed because you don’t have permission to access it.\nYou don’t have permission.\nTo view or change permissions, select the item in the Finder and choose File > Get Info.\n
```

Is this just timing? The test that fails about 50% of the time goes from creating the sim to deleting it. If I throw in a `sim.shutdown()`, which always fails but takes up time, then it seems to work well. If it's a timing thing, then it needs to be fixed somehow. I'm fine without this retry, it's just what worked to fix the tests.